x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -72,8 +73,27 @@ export function OverviewPage({ routeParams }: Props) {
     end: absoluteTime.end,
   });
 
+  // TODO: GET THE RIGHT CHECK
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] = hasAnyData
+    ? undefined
```

Comment:
**Home**

Reply:
This one is "kind of" right :) -- the "hasData" checks that get registered from each of our plugins only refer to whether there is any data right now (for a given time range), but not about whether the indices exist. We don't have that information here right now, so this is the best we can do. The "no data" screen would show up any time there is no data for any of the observability plugins in the given time range, if we go this way. (Alternatively we can log a ticket to provide "indicesExist" or soething like that with each of the registrations and then implement the no data screen for the home page after those all exist.)

Reply:
I logged this for us to do this at some point: https://github.com/elastic/kibana/issues/108312

Reply:
I'm honestly not sure which way is better, it probably depends on which less than ideal experience we want to have. Because of that, I'd probably lean toward leaving it how it is, but I'm not sure what kind of priority is being put on having these screens in place as much as possible.